### DISCUSSION TITLE
how to remove from firefox sidebar?

### DISCUSSION THREAD

**Discussion by of-me-is** [gildas-lormeau/SingleFile#1581] [Q&A]
i have over 15 extensions and this is the only one that shows up in sidebar under "sidebar extensions" and there's no way to remove it from there or drag it off or anything.
is there a way to do that?

**Comment by gildas-lormeau** [comment]
Unfortunately, this is almost impossible. The only way to achieve this today would be to deliver 2 separate extensions to the store, which is really not practical for me.

**Reply by of-me-is** [reply]
i see, thanks!
